frontend/react/src/components/map/ViewMap.jsx

```diff
@@ -0,0 +1,112 @@
+import React, {useState, useRef} from "react";
+import {
+    APIProvider,
+    Map,
+    AdvancedMarker,
+    Pin,
+    InfoWindow
+} from '@vis.gl/react-google-maps';
+import PropTypes from "prop-types";
+import classes from './ViewMap.module.css';
+
+// Define the props that should be passed to this component
+ViewMap.propTypes = {
+    markers: PropTypes.array
```

Comment:
Optional: You could include the shape of the elements in the array in the propTypes like you did in CourseView. Linting isn't complaining about this though, so I wouldn't consider it necessary.